src/_scss/layouts/search/header/_options.scss

```diff
@@ -25,8 +25,11 @@
         .download-button {
             @include searchHeaderDownloadButton;
         }
+
+        @import "../../../../../components/_comingSoon.scss";
```

Comment:
You can't have `@import` [inside mixins](http://sass-lang.com/documentation/file.SASS_REFERENCE.html#nested-import)